posts/2018-02-09-serverless-and-graphql-in-love.md

```diff
@@ -0,0 +1,560 @@
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Now, very reasonably, you are probably thinking, “Yeah, okay, Facebook is one thing; they have a giant engineering team. But for me, having only one API endpoint is too risky. What if it goes down? How do I handle that much load? What about security?”
+
+You are absolutely correct: with one HTTP endpoint, you need be entirely sure that endpoint never goes down and that it scales on demand.
+
+That’s where serverless comes in.
+
+## What is Serverless?
+
+Serverless has gained popularity over last few years, primarily because it gives developers flexibility.
+
+With Serverless comes the following:
+1. No server management (no need to manage any form of machine)
+2. Pay-per-execution (never pay for idle)
+3. Authentication and authorization at scale
+4. Function as a unit of application logic
+
+**Note:** I’m going to focus on AWS Lambda below, but know that you can use any serverless provider (Microsoft Azure, Google Cloud Functions, etc) with GraphQL.
+
+## What makes Serverless and GraphQL such a great fit?
+
+When moving to GraphQL, you suddenly rely on one HTTP endpoint to connect your clients to your backend services. Once you do decide to do that, you want this one HTTP endpoint to:
+
+1. be auto-scaling
+2. be reliable
+3. be fast
+4. have a small attack vector regarding security
+
+All these properties are fulfilled by a single AWS Lambda function in combination with API Gateway. It’s just a great fit! 
+
+In sum, powering your GraphQL endpoint with a serverless backend solves scaling and availability concerns outright, and it gives you a big leg up on security. It’s not even much code nor configuration. It takes only a few minutes to get to a production ready setup.
+
+I'd recommend you read [Jared's](https://twitter.com/ShortJared) [post](https://www.trek10.com/blog/a-look-at-serverless-graphql/) to get a better understanding of this relationship ;)
+
+## Serverless-GraphQL repository
+
+It’s pretty straightforward to get your HTTP endpoint up and running.
+
+I am happy to announce our Open Source Initiatives with [nikgraf](https://twitter.com/nikgraf) in [Serverless and GraphQL Repository](https://github.com/serverless/serverless-graphql). There, I walk through all the steps in detail. Go check it out!
+
+![alt text](https://user-images.githubusercontent.com/1587005/36035218-1c06763c-0d6b-11e8-996b-996243b0975f.png "Serverless and GraphQL Architecture")
+
+Repository comes in two flavor —
+
+1. API Gateway and Lambda Backend 
+
+ - API Gateway handles HTTP requests and responses and invokes the Lambda function in response to HTTP requests. 
```

Comment:
I would leave the description of the repository very simple, without detailed lists. Otherwise, it's going to get very difficult for the reader to hold so much information in their head. Less is more!

A simple (e.g.) "You can use the repository for API Gateway and Lambda backend solutions, or for an AppSync backend."

Reply:
make sense! I'll fix it